code/modules/reagents/bonsai.dm

```diff
@@ -0,0 +1,58 @@
+/** Club's special Bonsai tree, so they can operate somewhat without a garden.
```

Comment:
While this is a cool artifact, the club has a private garden of their own and the public garden.

Still, this is a neat anomoly and fits thematically.